dask/dataframe/tests/test_dataframe.py

```diff
@@ -66,6 +66,7 @@
 except ImportError:
     ArrowNotImplementedError = RuntimeError  # some unrelated error to make pytest pass
 
+dask_expr_enabled = dd._dask_expr_enabled()
```

Comment:
nit: Use capital letters to highlight constant

```suggestion
DASK_EXPR_ENABLED = dd._dask_expr_enabled()
```